docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -268,6 +268,63 @@ _Triggers the **[OwnershipTransferred](#ownershiptransferred)** event after succ
 Leaves the contract without an owner. Once ownership of the contract is renounced, it won't be possible to call the functions restricted to the owner only.
 :::
 
+### batchCalls
+
+:::info
+
+Check the [**batchCalls(..)**](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-0-ERC725Account.md#batchcalls) function specification in [**LSP0-ERC725Account Standard**](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-0-ERC725Account.md) in the **LIP repository**.
+
+:::
+
+```solidity
```

Comment:
same comments as in the PR for the LIP repo: https://github.com/lukso-network/LIPs/pull/189